rewards/boost/boost_utils.py

```diff
@@ -44,9 +45,16 @@ def calc_boost_balances(
     native = Counter()
     non_native = Counter()
 
+    nft_balances = nft_snapshot_usd(chain, block)
+    console.log(nft_balances)
```

Comment:
Let's either get rid of this or make it more descriptive if we want to capture it.  Just logging a random number doesn't do us any good reviewing logs.